ssg/build_templates.py

```diff
@@ -73,7 +73,6 @@ def __init__(self, env_yaml):
             "audit_rules_login_events.csv":  AuditRulesLoginEventsGenerator(),
             "audit_rules_privileged_commands.csv":  AuditRulesPrivilegedCommandsGenerator(),
             "audit_rules_usergroup_modification.csv":  AuditRulesUserGroupModificationGenerator(),
-            "audit_rules_usergroup_modification.csv":  AuditRulesUserGroupModificationGenerator(),
```

Comment:
Is this change really intended?

Reply:
Yes, is a duplicated line. Unless is has some shady meaning.